tox.ini

```diff
@@ -12,7 +12,6 @@ install_command = python -m pip install --only-binary :all: {opts} {packages}
 setenv = PYTHONDEVMODE = 1
 commands =
     coverage run -m pytest --doctest-modules --pyargs cutadapt tests
-    coverage combine -q
```

Comment:
Can you say why these changes were necessary? `concurrency = multiprocessing` is (or at least was) necessary to get coverage in subprocesses. (Which then in turned required `coverage combine`.)